src/actions/authActions.js

```diff
@@ -3,14 +3,22 @@ export const USER_LOGGED_IN = 'USER_LOGGED_IN'
 export const USER_LOGGED_OUT = 'USER_LOGGED_OUT'
 export const USER_LOGIN_ERROR = 'USER_LOGIN_ERROR'
 
+var registerAuthenticatedId = (signInId) => {
```

Comment:
If this is used only within the userLoggedIn function, it might be best to define it within that function. 

Also, prefer const over var for variables when you don't intend for them to be changed after initially being defined (which is generally the case for functions), since that reduces the potential complexity of the code.